crates/client/rpc-core/src/lib.rs

```diff
@@ -47,6 +47,16 @@ pub struct DeclareV0Result {
     pub class_hash: ClassHash,
 }
 
+/// CustomDeclareV0Transaction
+/// This will be given as an input to the rpc body
+///
+/// **declare_transaction** : DeclareTransactionV0V1 struct constructed before calling.
+///
+/// **program_vec** : Program vector is program from `ClassInfo` struct converted into Vector.
```

Comment:
`program_bytes`

Reply:
do you want me to rename `program_vec` to `program_bytes` ?

Reply:
vec is not really descriptive, bytes add a bit of information. So yes.

Reply:
done